Fix: Unify button gradient and add hover animation #<issue-number>

khusshii26 opened the pull request:
This PR addresses the UI enhancement issue by unifying the gradient button styles across the application. A reusable GradientButton component has been created to ensure consistency in design. Additionally, smooth hover animations have been added for better user interaction and experience.

 1) Created a new GradientButton component in components/ui/GradientButton.tsx

  2)  Replaced all standalone gradient buttons with the unified component

   3) Added hover animation using Tailwind CSS transitions

ankitmodanwall commented:
@khusshii26 provide screenshot or loom video once.

mansiruhil commented:
<img width="1366" height="633" alt="home" src="https://github.com/user-attachments/assets/651e35ea-5a81-4d7c-a457-0005d670a31d" />
- update the button gradient as current colors blend into the black bg
- not include glow on the text "welcome .."

khusshii26 commented:
https://github.com/user-attachments/assets/04033f5b-5941-4f03-9570-70e450cd8a0f

I've updated the gradient button and removed the glow from the "Welcome" text as discussed.
Also, I’ve recorded a short video demo with the current changes.

Could you please check and let me know if it  looks good to you, or if there’s anything else you’d like me improve?

mansiruhil commented:
can you change color? like semi transparent white  gives a frosted glass effect and background shows through nicely
or white ?
and button size of explore and about failuforward should be same 
and similarly color for setbacks, network, daily

mansiruhil commented:
done?

khusshii26 commented:
working on it
